app/views/devise/sessions/new.html.erb

```diff
@@ -1,16 +1,20 @@
-<h2>Log in</h2>
 
+
+<div class="govuk-grid-row">
+<div class='govuk-grid-column-two-thirds govuk-!-margin-top-3 govuk-!-margin-bottom-3'>
+<h1 class="govuk-heading-l">Sign in</h2>
 <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  <div class="field govuk-form-group">
+    <%= f.label :email, class: 'govuk-label' %>
```

Comment:
I think we can use our components from `govuk_design_system_formbuilder` (see https://govuk-form-builder.netlify.app/form-elements/text-input/) as we do in our other forms .e.g something like:

```rb
<%= f.govuk_email_field :email, label: {text: 'Email'}, width: 'one-half' %>
```